docs/stopwatch.adoc

```diff
@@ -3,18 +3,40 @@
 :xp-demo-dir: ../src/demo/java
 :demo: {xp-demo-dir}/org/junitpioneer/jupiter/StopwatchExtensionDemo.java
 
-== Introduction
-
-Test shall be fast.
-To get a feeling how fast or slow a specific test is, the `@Stopwatch` annotation can be used.
-It will measure elapsed time and pass the result to the `https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/TestReporter.html[TestReporter]`, which will publish it.
+Annotating a test with `@Stopwatch` will measure the time it takes to execute and report the result to the `https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/TestReporter.html[TestReporter]`.
```

Comment:
```suggestion
Annotating a test with `@Stopwatch` will measure the time it takes to execute and report the result to the https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/TestReporter.html[`TestReporter`].
```

Reply:
Your approach is also valid, but we use link[`code`] elsewhere (in the same documentation even).

Reply:
True!